src/test/java/net/sf/jabref/logic/util/io/UtilFindFileTest.java

```diff
@@ -21,24 +23,36 @@
  */
 public class UtilFindFileTest extends FileBasedTestCase {
 
-
     @Test
-    @Ignore(value = "works on windows but not on linux")
     public void testFindAssociatedFiles() {
+        Path rootPath = root.toPath().toAbsolutePath();
         Collection<BibEntry> entries = Collections.singletonList(entry);
         Collection<String> extensions = Arrays.asList("jpg", "pdf");
-        Collection<File> dirs = Arrays.asList(new File(root.getAbsoluteFile() + "/pdfs/"),
-                new File(root.getAbsoluteFile() + "/graphicsDir/"));
+        Collection<File> dirs = Arrays.asList(rootPath.resolve("graphicsdir").toFile(),
+                rootPath.resolve("pdfs").toFile());
 
         Map<BibEntry, List<File>> results = FileUtil.findAssociatedFiles(entries, extensions, dirs);
 
         Assert.assertEquals(2, results.get(entry).size());
+        Assert.assertTrue(results.get(entry)
+                .contains(rootPath.resolve(Paths.get("graphicsDir", "subDir", "HipKro03test.jpg")).toFile()));
+        Assert.assertFalse(results.get(entry)
+                .contains(rootPath.resolve(Paths.get("graphicsDir", "subDir", "HipKro03test.png")).toFile()));
```

Comment:
@oscargus  I think I found the error! Case Sensitivy. Windows is case insenstive, but Linux not!
`graphicsDir `vs g`raphicsdir`